translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+        The move should be made along this axis, forward or backwards.
+        There can be no opponent nor holes in the phalanx.</source>
+        <target>Bravo !
+        Les pièces déplacées doivent être horizontalement, verticalement, ou diagonalement alignées.
+        Le déplacement doit se faire le long de cette ligne, en avant ou en arrière.
+        Il ne peut y avoir ni pièces adverses ni trous dans la phalange.</target>
+      </trans-unit>
+      <trans-unit id="1735581478820014059" datatype="html">
+        <source>In order to capture pieces of the opponent:
+        &lt;ol&gt;
+            &lt;li&gt;It must be aligned with the phalanx you are moving.&lt;/li&gt;
+            &lt;li&gt;It must be strictly shorter than your phalanx.&lt;/li&gt;
+            &lt;li&gt;The first piece of your phalanx should land on the first piece of the opponent&apos;s phalanx that you want to capture.&lt;/li&gt;
+        &lt;/ol&gt;&lt;br/&gt;
+        Capture a phalanx.</source>
+        <target>Pour capturer une phalange de l'adversaire :
+        &lt;ol&gt;
+            &lt;li&gt;Il faut que celle-ci soit alignée avec la phalange en déplacement.&lt;/li&gt;
+            &lt;li&gt;Qu&apos;elle soit strictement plus courte.&lt;/li&gt;
+            &lt;li&gt;Que la première pièce de votre phalange atterrisse sur la première pièce rencontrée de la phalange à capturer.&lt;/li&gt;
+        &lt;/ol&gt;&lt;br/&gt;
+        Capturez la phalange.</target>
+      </trans-unit>
+      <trans-unit id="8213276201685541009" datatype="html">
+        <source>Congratulations, you made it!
+        Note that the diagonal phalanx was not aligned with your phalanx, hence even if it is longer, this does not prevent you from capturing some of its pieces on another alignment.</source>
+        <target>Bravo, vous avez réussi.
+        Constatez que la phalange diagonale n'étant pas alignée avec la notre, sa longueur supérieur n'empêche pas de capturer ses pièces dans un autre alignement.
+        </target>
+      </trans-unit>
+      <trans-unit id="4418812710815829575" datatype="html">
+        <source>Failed, you have not captured the phalanx.</source>
+        <target>Raté, vous n&apos;avez pas capturé la phalange.</target>
+      </trans-unit>
+      <trans-unit id="7226802484619632640" datatype="html">
+        <source>You can only capture when 4 or more of your pieces are aligned, and it is not the case.</source>
+        <target>Une capture ne peut que se faire si 4 pièces de votre couleurs sont alignées, ce n&apos;est pas le cas.</target>
+      </trans-unit>
+      <trans-unit id="6918785733984182442" datatype="html">
+        <source>You must select a valid capture that contains 4 pieces or more.</source>
+        <target>Veuillez choisir une capture valide qui contient 4 pièces ou plus.</target>
+      </trans-unit>
+      <trans-unit id="8805744649589430247" datatype="html">
+        <source>There are captures remaining.</source>
```

Comment:
There are possible captures remaining. 
ou
There are still possible captures to be done.
 ?

Reply:
second choix, plus explicite